tests/components/remote/common.py

```diff
@@ -24,19 +24,23 @@
 from homeassistant.loader import bind_hass
```

Comment:
We're trying to get rid of these files, not add to it. The only reason they exist is an old legacy design decision that has since been reverted. 

The main issue with them is that they don't block. So instead of using such methods, consider just calling `await hass.services.async_call(…, blocking=True)`

Reply:
i believe frenck had me add the blocking=True for that reason.

I can unserstand the desire to reduce test utils but having these would save many lines of such boiler that are normally used in such tests. I feel its likely better to have these around than the individual test versions getting out of date or containing varied logic but obv thats just my thought on it :)

Frenck had done this for media player and it saved 90 lines of duplication in directv alone and likely the same in roku.

Reply:
I can see your point @ctalkington, but in reality, you can call the service directly. Which is not much more code compared to what you already have now.

Honestly, I didn't know. I just converted all demo tests to async and added more of these as well. So, I learned 🤷‍♂

Reply:
created https://github.com/home-assistant/core/pull/33793 and will close this. I  feel like media player has more aspects/services to cover and those should stay and be allowed for use but ill leave that to @balloob to decide